src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {
```

Comment:
It's unnecessary inheritance.  Please, follow the rule: "is-a, has-a". 

For this task, you shouldn't extend Lottery class from Ball class